README.md

```diff
@@ -116,7 +116,7 @@ engine = create_engine(
     connect_args={
       "session_properties": {'query_max_run_time': '1d'},
       "client_tags": ["tag1", "tag2"],
-      "experimental_python_types": True,
+      "legacy_primitive_types": True,
```

Comment:
Remove these entries, these code parts tend to be copied by users and we would rather not let them set this to True.

Reply:
removed